cxx/Makefile

```diff
@@ -37,8 +37,8 @@ LTEST		= $(wildcard $(TEST_DIR)/*.cc)
 %.out : %.cc $(LOBJECT)
 	$(CXX) $(CXXFLAGS) $(INCFLAGS) -o $@ $(LOBJECT) $<
 
-.PHONY: tests
-tests: $(LTEST:.cc=.out) hirm.out
+.PHONY: integration_tests
```

Comment:
Make sure to also update the README with the new command for running the integration tests.

Reply:
Done.